tests/unit/package_managers/test_general.py

```diff
@@ -145,10 +146,30 @@ def test_extract_git_info(url: str, nonstandard_info: Any) -> None:
 
 
 @pytest.mark.asyncio
-async def test_async_download_binary_file(tmp_path: Path, caplog: pytest.LogCaptureFixture) -> None:
+@pytest.mark.parametrize(
+    "timeout",
+    [
+        pytest.param(300, id="default_timeout"),
```

Comment:
Why do we need this parametrization? The codepath is the same, it has no effect on test